app/controllers/govuk_publishing_components/applications_page_controller.rb

```diff
@@ -0,0 +1,77 @@
+module GovukPublishingComponents
+  class ApplicationsPageController < GovukPublishingComponents::ApplicationController
+    def show
+      applications = %w[
+        account-api
+        collections
+        collections-publisher
+        content-data-admin
+        content-publisher
+        content-tagger
+        datagovuk_find
+        email-alert-frontend
+        feedback
+        finder-frontend
+        frontend
+        government-frontend
+        govspeak
+        govspeak-preview
+        govuk-chat
+        govuk-developer-docs
+        local-links-manager
+        manuals-publisher
+        maslow
+        places-manager
+        publisher
+        release
+        search-admin
+        search-v2-evaluator
+        service-manual-publisher
+        short-url-manager
+        signon
+        smart-answers
+        specialist-publisher
+        static
+        support
+        transition
+        travel-advice-publisher
+        whitehall
```

Comment:
Should we include GOV.UK Chat in the list? https://github.com/alphagov/govuk-chat